src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -751,6 +754,48 @@ void ov::intel_cpu::MKLDNNInferRequest::SetBlob(const std::string& name, const I
     }
 }
 
+void ov::intel_cpu::MKLDNNInferRequest::SetBlobsImpl(const std::string& name, const InferenceEngine::BatchedBlob::Ptr& batched_blob) {
+    _batched_inputs[name] = batched_blob;
+}
+
+void ov::intel_cpu::MKLDNNInferRequest::convertBatchedInputBlob(const std::string& name, const InferenceEngine::BatchedBlob::Ptr& batched_blob) {
+    auto tmp_desc = batched_blob->getBlob(0)->getTensorDesc();
+    tmp_desc.getDims()[0] = batched_blob->size();
+    auto blockingDims = tmp_desc.getBlockingDesc().getBlockDims();
+    blockingDims[0] = batched_blob->size();
+    auto blockingDesc = InferenceEngine::BlockingDesc(blockingDims, tmp_desc.getBlockingDesc().getOrder());
+    auto batched_desc = InferenceEngine::TensorDesc(tmp_desc.getPrecision(), tmp_desc.getDims(), blockingDesc);
+
+    InferenceEngine::MemoryBlob::Ptr mem_blob = std::dynamic_pointer_cast<InferenceEngine::MemoryBlob>(make_blob_with_precision(batched_desc));
```

Comment:
Is it possible to avoid calling of 'make_blob_with_precision' on every inference? If SetBlob was called once, there probably no need to perform reallocation on next inference. Default implementation has similar issue which should be fixed as well (maybe not in this PR)